Sources/Plasma/FeatureLib/pfAnimation/pfObjectFlocker.cpp

```diff
@@ -435,32 +435,26 @@ hsPoint3 pfBoidGoal::PredictFuturePosition(const float predictionTime)
 ///////////////////////////////////////////////////////////////////////////////
 
 pfBoid::pfBoid(pfProximityDatabase& pd, pfObjectFlocker *flocker, plKey &key)
+    : fProximityToken(), fFlockerPtr(flocker), fObjKey(key), fWanderSide(), fWanderUp()
```

Comment:
This is not your bug, but attempting to initialize `fObjKey` with an non-const reference will error on clang. Ideally, we want a copy of `key` and to `std::move(key)` into `fObjKey`.